### ISSUE TITLE
Issues with training and testing on the MSEC dataset

### ISSUE THREAD

**Issue by id4su** [onpix/LCDPNet#23]
Hello, when I train on the dataset you proposed, everything goes relatively well. However, I have the following problem when I train on the MSEC dataset, can you help me? Thanks!
![image](https://github.com/onpix/LCDPNet/assets/87803154/008e5cbe-37f9-40f1-9a96-b5bcacd1e576)
I'm sure my path settings are correct.
![image](https://github.com/onpix/LCDPNet/assets/87803154/754e94b3-06fa-4f5c-a570-d36298eb7848)
Actually, I found that in the training set, validation set and testing set of MSEC, five input maps correspond to one ground truth image, how can I solve this problem so that I can train and test normally.

**Comment by onpix** [comment]
Please try this setting:

```yaml
# @package ds
class: img_dataset
name: MSEC
input:
  - /public/hywang26/ds/msec/train/INPUT_IMAGES/*_0.JPG
  - /public/hywang26/ds/msec/train/INPUT_IMAGES/*_N1.JPG
  - /public/hywang26/ds/msec/train/INPUT_IMAGES/*_N1.5.JPG
  - /public/hywang26/ds/msec/train/INPUT_IMAGES/*_P1.JPG
  - /public/hywang26/ds/msec/train/INPUT_IMAGES/*_P1.5.JPG
GT:
  - /public/hywang26/ds/msec/train/GT_IMAGES/*
  - /public/hywang26/ds/msec/train/GT_IMAGES/*
  - /public/hywang26/ds/msec/train/GT_IMAGES/*
  - /public/hywang26/ds/msec/train/GT_IMAGES/*
  - /public/hywang26/ds/msec/train/GT_IMAGES/*
```

which will extend GT 5 times, one for each input image subset.
